src/systems/physics/Physics.cc

```diff
@@ -373,6 +382,7 @@ class ignition::gazebo::systems::PhysicsPrivate
           physics::World,
           MinimumFeatureList,
           CollisionFeatureList,
+          ContactFeatureList,
```

Comment:
If adding this, `CollisionFeatureList,` can be removed

Reply:
Following that logic, `MinimumFeatureList` should have been removed before, I'm leaving them all just in case

Reply:
I agree with @Lobotuerk, I think we want to support physics engines that support `CollisionFeatureList`, but not `ContactFeatureList`.